ansible_collections/arista/avd/plugins/action/cv_workflow.py

```diff
@@ -370,26 +352,37 @@ async def build_object_for_device(
         # !      - name: peer_device_interface
         # !        value: Ethernet3
         all_interface_tags = get(structured_config, "metadata.cv_tags.interface_tags", default=[])
-        interface_tag_objects = [
-            CVInterfaceTag(
+        # Build tuple of interface tag objects
+        avd_interface_tag_objects = tuple(
+            AvdInterfaceTag(
                 label=interface_tag["name"],
                 value=interface_tag["value"],
-                device=device_object,
                 interface=one_interface_tags["interface"],
             )
             for one_interface_tags in all_interface_tags
             if "interface" in one_interface_tags and "tags" in one_interface_tags
             for interface_tag in one_interface_tags["tags"]
             if "name" in interface_tag and "value" in interface_tag
-        ]
+        )
 
         # Build WAN metadata object for this device.
-        cv_pathfinder_metadata_objects = []
+        avd_cv_pathfinder_metadata_object = AvdPathfinderMetadata(metadata={})
```

Comment:
```suggestion
        avd_cv_pathfinder_metadata_object = AvdPathfinderMetadata(metadata= get(structured_config, "metadata.cv_pathfinder", default={}))
```
Then you can remove the if statement below.